raw_data/preprocessed/africa/botswana_english.txt

```diff
@@ -0,0 +1,107 @@
+
+Conduct of public hearing
+(13) [1] The competent authority may hold a public hearing if [a] after examining the statement, the competent authority is of the opinion that the activity is of such a nature that the public should have the opportunity to make submissions or comments at a public hearing; or [b] the public concern over the activity is that the activity may have a significant adverse impact on the environment. [2] The Minister may, in consultation with the competent authority, prescribe the procedure for conducting a public hearing. [3] The competent authority shall, after a public hearing, consider the findings in determining the adequacy of the environmental impact statement.
+
+Approval of statement
+(14) [1] Where the competent authority's review of the statement is complete, the competent authority shall [a] grant authorisation to the developer, with such terms and conditions as the competent authority considers necessary, where (i) the competent authority is satisfied that the statement sufficiently identifies the environmental impact likely to be caused, and [ii]  the prescribed mitigation measures in the statement, to avert or minimise the potential adverse environmental impact, are effective and sufficient; or [b] where the competent authority is of the view that (i) the statement does not sufficiently identify the environmental impact likely to be caused, or [ii]  the prescribed mitigation measures to avert or minimise the potential adverse environmental impact, are insufficient and ineffective, afford the developer an opportunity to comply with the provisions of the Act; or [c] reject the statement, where the developer fails to satisfy the conditions under paragraph [b]. [2] In deferring or rejecting the statement, the competent authority shall furnish the applicant with a written statement of the reasons for its decision. [3] An authorisation issued under subsection [1][a] shall be in the prescribed form and shall include [a] what has been decided; [b] why the decision has been made; and [c] how the statement influenced the decision. [4] The competent authority may, in issuing an authorisation, prescribe, in writing, specific requirements for monitoring during and after implementation of the proposed activity, by the technical departments, local authorities and the developer.
```

Comment:
Please replace the two instances of `(i)` with `(ii)`.

Reply:
Do you mean replace (i) with [i]?